requirements.txt

```diff
@@ -10,3 +10,5 @@ PyYAML==6.0.1
 six==1.16.0
 tzdata==2023.4
 urllib3==1.26.18
+fastapi==0.110.0
+python-multipart==0.0.9
```

Comment:
Tried installing this latest requirements and got error below when trying to run: 

```
(venv3) prubenda@prubenda1-mac orion % orion --debug
Traceback (most recent call last):
  File "/Users/prubenda/PycharmProjects/pplRepos/shashank/orion/venv3/bin/orion", line 5, in <module>
    from orion import cli
  File "/Users/prubenda/PycharmProjects/pplRepos/shashank/orion/venv3/lib/python3.10/site-packages/orion.py", line 9, in <module>
    import uvicorn
ModuleNotFoundError: No module named 'uvicorn'
```

Reply:
will add uvicorn to requirements.txt, also planning on integrating pdm so we don't run into requirement issues.

Reply:
Please use `orion cmd-mode` or `orion daemon-mode`, I added these new commands to differentiate between the two.